arbor/backends/gpu/matrix_state_interleaved.hpp

```diff
@@ -267,10 +268,10 @@ struct matrix_state_interleaved {
     //   dt_intdom         [ms]     (per integration domain)
     //   voltage           [mV]     (per compartment)
     //   current density   [A.m^-2] (per compartment)
-    void assemble(const_view dt_intdom, const_view voltage, const_view current) {
+    void assemble(const_view dt_intdom, const_view voltage, const_view current, const_view conductivitt) {
         assemble_matrix_interleaved
             (d.data(), rhs.data(), invariant_d.data(),
-             voltage.data(), current.data(), cv_capacitance.data(), cv_area.data(),
+             voltage.data(), current.data(), conductivitt.data(), cv_capacitance.data(), cv_area.data(),
```

Comment:
`conductivtt` or `conductivity`?

Reply:
Yeah, that was an interesting typo that somehow was correctly propagated and compiled.